packages/elements-react/src/theme/default/components/form/social.tsx

```diff
@@ -47,6 +47,12 @@ export function DefaultButtonSocial({
     node_type: _ignoredNodeType,
     type: _ignoredType,
     name: _ignoredName,
+    onclickTrigger: _ignoredOnClickTrigger,
```

Comment:
@jonas-jonas should we just put these in a list and build a general cleaner? A lot of components have these issues